### ISSUE TITLE
Merge tn-en into en-tn and Make RC

### ISSUE THREAD

**Issue by jag3773** [unfoldingWord-dev/door43.org#343]
## Story

As a *user* I want to a consistent & accurate source location for tN so that I know where to find the latest and fork if needed.

## Notes

Blocked by #88 

Before we moved tN into Gogs following the new RC format, we had copied it over to [tn-en](https://git.door43.org/Door43/tn-en) to try to make revision history more viewable.  Now we have two repos purporting to hold the source for English tN.

The [en-tn](https://git.door43.org/Door43/en-tn) repo needs to be the master and should have the revision history of the previous repo if possible.  In other words, it would be great if we could have all the tagged versions from tn-en merged into en-tn so that there is one place to see all the revisions.

In addition, the en-tn repo is not a valid RC right now, we should take this opportunity and make it so.

End goal: [en-tn](https://git.door43.org/Door43/en-tn) is a valid RC and it has versions 1-4 tagged correctly as Releases.

**Comment by neutrinog** [comment]
@jag3773 there's this PR that's a year old on tn-en. I'm 99.9999% certain we don't want to accept it but wanted to double check before we nuke everything https://git.door43.org/Door43/tn-en/pulls/1.

**Comment by neutrinog** [comment]
[tn-en](https://git.door43.org/Door43/tn-en) has been merged into [en-tn](https://git.door43.org/Door43/en_tn). I have not converted it into an RC yet.

**Comment by jag3773** [comment]
Yes, we want a clear revision history.

**Comment by neutrinog** [comment]
we will convert these to RCs.

**Comment by neutrinog** [comment]
these have been converted and will be included with the migration from dokuwiki.
